tests/conda_env/test_create.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function
```

Comment:
This file was based on conda's test_create.py
Please let me know if this should be different